### ISSUE TITLE
LICENCE -> LICENSE?

### ISSUE THREAD

**Issue by passy** [vanniktech/gradle-maven-publish-plugin#252]
Hey!

Firstly, thanks a ton for your plugin. This has been incredibly helpful in moving off JCenter.

With the upgrade to 0.15 or 0.15.1, our license field in the uploaded POM was no longer getting populated, failing the Maven Central sync requirements.

After a while, I noticed that you require the fields `"POM_LICENCE_*"` instead of the more common spelling `"POM_LICENSE_*"`.

I was wondering if you could support the previous variant again?

**Comment by gabrielittner** [comment]
Which version did you use before? It looks like we were always expecting `LICENCE` 
https://github.com/vanniktech/gradle-maven-publish-plugin/blob/0.14.0/src/main/kotlin/com/vanniktech/maven/publish/MavenPublishPom.kt#L43-L45
https://github.com/vanniktech/gradle-maven-publish-plugin/blob/0.1.0/src/main/groovy/com/vanniktech/maven/publish/MavenPublishPlugin.groovy#L66-L68

**Comment by passy** [comment]
Fascinating. We've had the `LICENSE` properties in our global `gradle.properties` for a while and it broke around the time we upgraded to 0.15. We have weekly releases and the upgrade was somewhere in between. Thanks for the pointers. I'll try and track this down on Monday.

**Comment by gabrielittner** [comment]
I'm still open for also supporting LICENSE, that's also how I would spell it, I never even noticed that we spell it differently. Just curious on why it was working before 0.15

**Comment by passy** [comment]
You're absolutely right. I've downgraded all the way to 0.13 which was the first one we pulled into the project and it doesn't populate the fields from `LICENSE_` fields either. My only guess is that some other dependency had the side-effect of populating the corresponding `LICENSE` fields or we still had an old-style `publishing {}` config around that read from the old fields.

Sorry for the wrong report, but I agree that it would be quite nice to support the American spelling here for consistency even if it just served as a fallback.
